src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -109,12 +112,36 @@ void GetInteriorVehicleDataRequest::Execute() {
     RemoveExcessiveSubscription();
   }
 
-  (*message_)[app_mngr::strings::msg_params][app_mngr::strings::app_id] =
-      app->app_id();
+  auto app_unsubscribes =
+      !((*message_)[app_mngr::strings::msg_params][message_params::kSubscribe]
```

Comment:
@LuxoftAKutsan please notice that **_subscribe_** is not mandatory param

Reply:
@LitvinenkoIra fixed ee8a3940c53dc8a6ed6f4949c19ea62a0a658c98